src/NUnitFramework/framework/Constraints/Tolerance.cs

```diff
@@ -169,10 +169,14 @@ public Tolerance Ticks
         public object Amount { get; }
 
         /// <summary>
-        /// Returns true if the current tolerance has not been set or is using the .
+        /// Returns true if the current tolerance has not been set or is using the default.
         /// </summary>
         public bool IsUnsetOrDefault => Mode == ToleranceMode.Unset;
 
+        /// <summary>
+        /// Returns true if the current tolerance varies from exact and default. Indicating tolerance needs processing.
+        /// </summary>
+        public bool HasVariance => Mode != ToleranceMode.Unset && this != Exact && this != Default;
```

Comment:
Default should not be necessary as that has mode `Unset`.
Also, the match on `Exact` is a reference comparison, and won't match `Within(0)`.
But maybe we don't want to, if they want a perfect match, don't specify `Within` at all.

Reply:
What is the reason for  `HasVariance` being slightly different then `!IsUnsertOrDefault`?
What it your use case?